### PR TITLE
5151/fix/slick accessibility update quick fix

### PR DESCRIPTION
**Pull request by Realmbird**
Closes #5151
Fixes PR https://github.com/internetarchive/openlibrary/pull/5151

As Accessibility already exists in slick carousel. This PR removes the default slick accessibility settings as it causes a glitch where after user clicks the next arrow button the carousel doesn't go to the next books in the carousel and there was another glitch were after clicking arrow on slick even though carousel went to next books keyboard reader skips past carousel.

### Technical

Simply removes default accessibility code from slick that causes bugs when combined with OL accessibility

### Testing

1. docker compose up (make image)
2. docker compose run --rm home npm run build-assets (change js)
3. http://localhost:8080/works/OL36887156W/Short_Story_Collection_Vol_078?edition=shortstory078_1902_librivox
4. press tab until carousel
5. Observe how tab on next or prev arrow causes carousel to to scroll and allow user to go to next books with just tab.
6. After going through carousel use shift+tab to check going through carousel backwards

Notes while it looks like a error when interacting with arrow going to middle book you can see how it actually skipped the books that the user already traversed through. As there is not a enough books for a whole row so after interacting with arrow user goes to next book in tab reader.

### Screenshot

Before weird interaction next button doesn't allow carousel to scroll with tab reader
[Initial Slick not working.webm](https://github.com/internetarchive/openlibrary/assets/56459277/87f26e1f-e717-4e1a-8e2f-171ec5cbfbde)
After use inspect to observe how it follows order of data-slick-index
[Screencast from 05-04-2024 12:33:13 PM.webm](https://github.com/internetarchive/openlibrary/assets/56459277/0e6805f4-5e5b-4a1e-84a7-20fe24465c64)

### Stakeholders

@mekarpeles @RayBB

### CODE REVIEW

**Comment by Realmbird** [comment]
Please remove changes to buttonCta.less made mistake not sure how to remove that

**Comment by Realmbird** [comment]
For testing use this

File: openlibrary/plugins/openlibrary/js/carousel/Carousel.js [hunk 1]
```diff
@@ -174,10 +175,11 @@ export class Carousel {
                 </div>
             </div>`);
         $el.find('.bookcover').attr('title', work.title);
+        // $el.find('.slick-prev slick-arrow').attr('aria-hidden', false)
         $el.find('.cta-btn')
             .attr('title', `${cta}: ${work.title}`)
             .attr('data-ocaid', ocaid)
-            .attr('href', url);
+            .attr('href', url)
```
Comment (RayBB) [inline openlibrary/plugins/openlibrary/js/carousel/Carousel.js:182]:
```suggestion
            .attr('href', url);
```

**Review by RayBB — CHANGES_REQUESTED** [review]
removing the changes as you requested

**Comment by RayBB** [comment]
@Realmbird sorry for the delay in this.
I see this is a tricky one to test.
On chrome at least, it seems this is causing some strange issue with the covers.
On firefox the behavior is totally different not selecting books at all.

I ready through the original PR more closely again and I'm still not totally sure what needs to happen here or if perhaps the issue has been resolved.

I will try to bring this up in the call tomorrow.


On chrome:
https://github.com/internetarchive/openlibrary/assets/921217/92cf8395-990a-4901-a010-be3d28852664

**Comment by RayBB** [comment]
Closing per https://github.com/internetarchive/openlibrary/pull/5151#issuecomment-2100515629

In short, we don't have the experience right now to be sure the change we are making is the right thing so we opened a ticket to investigate and will hold off for now.

Thanks for pushing this forward to the point where we can understand what our next steps are and sorry that this one won't end up being merged!

**Pull request closed without merging**